.cloud-build/execute_changed_notebooks_cli.py

```diff
@@ -47,6 +50,12 @@ def str2bool(v):
     required=False,
     default=100,
 )
+parser.add_argument(
+    "--build_id",
+    type=str,
+    help="The build id (which may be cloud run specified or user explicit.",
+    default=RESULTS_FILE
```

Comment:
RESULTS_FILE no longer makes sense to be a `build_id`, consider removing and making this required.